<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PVP_Settings</class>
 <widget class="QDialog" name="PVP_Settings">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PVP_Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	background-color:#ffffff;
 	border:none;
	radius:20px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="tone_Setting">
    <property name="geometry">
     <rect>
      <x>747</x>
      <y>308</y>
      <width>179</width>
      <height>38</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>38</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
	background:transparent;
	border: 1px solid rgba(0,0,0,0.2);
	padding:8px;
}</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>正常</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>低调</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>浮夸</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="num_Setting">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>428</y>
      <width>80</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
	background:transparent;
	border: 1px solid rgba(0,0,0,0.2);
	padding:8px;
    padding-right:0px;

}</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="move_Setting">
    <property name="geometry">
     <rect>
      <x>747</x>
      <y>368</y>
      <width>179</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
	background:transparent;
	border: 1px solid rgba(0,0,0,0.2);
	padding:8px;
    padding-right:0px;

}</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <item>
     <property name="text">
      <string>正常</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>浮夸</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>低调</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="Name_Edit">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>122</y>
      <width>275</width>
      <height>44</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
	background:transparent;
	border: 1px solid rgba(0,0,0,0.3);
	padding:5px;
    border-radius:8px;
}</string>
    </property>
    <property name="placeholderText">
     <string>请输入名称</string>
    </property>
   </widget>
   <widget class="QComboBox" name="emoji_Setting">
    <property name="geometry">
     <rect>
      <x>747</x>
      <y>428</y>
      <width>179</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
	background:transparent;
	border: 1px solid rgba(0,0,0,0.2);
	padding:8px;
    padding-right:0px;

}</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <item>
     <property name="text">
      <string>经典表情</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>更多...</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="NO4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>578</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
    spacing: 0px;
}

QRadioButton::indicator {
    width: 31px;
    height: 31px;
}
QRadioButton::indicator:checked {
  image: url(:/Resource/UI_Design/RadioButton2.png);
}
QRadioButton::indicator:unchecked {
    image: url(:/Resource/UI_Design/RadioButton.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm">
    <property name="geometry">
     <rect>
      <x>714</x>
      <y>610</y>
      <width>138</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Resource/UI_Design/#Block.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../Resource/Gobang.qrc">
      <normaloff>:/Resource/UI_Design/pvpSettings_Save.png</normaloff>:/Resource/UI_Design/pvpSettings_Save.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>138</width>
      <height>70</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="NO2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>316</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
    spacing: 0px;
}

QRadioButton::indicator {
    width: 31px;
    height: 31px;
}
QRadioButton::indicator:checked {
  image: url(:/Resource/UI_Design/RadioButton2.png);
}
QRadioButton::indicator:unchecked {
    image: url(:/Resource/UI_Design/RadioButton.png);
}
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="NO3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>443</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
    spacing: 0px;
}

QRadioButton::indicator {
    width: 31px;
    height: 31px;
}
QRadioButton::indicator:checked {
  image: url(:/Resource/UI_Design/RadioButton2.png);
}
QRadioButton::indicator:unchecked {
    image: url(:/Resource/UI_Design/RadioButton.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="NO1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>189</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
    spacing: 0px;
}

QRadioButton::indicator {
    width: 31px;
    height: 31px;
}
QRadioButton::indicator:checked {
  image: url(:/Resource/UI_Design/RadioButton2.png);
}
QRadioButton::indicator:unchecked {
    image: url(:/Resource/UI_Design/RadioButton.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>955</x>
      <y>610</y>
      <width>138</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Resource/UI_Design/#Block.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../Resource/Gobang.qrc">
      <normaloff>:/Resource/UI_Design/pvpSettings_Cancel.png</normaloff>:/Resource/UI_Design/pvpSettings_Cancel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>138</width>
      <height>70</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="auto_counter">
    <property name="geometry">
     <rect>
      <x>957</x>
      <y>532</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="Provocative">
    <property name="geometry">
     <rect>
      <x>946</x>
      <y>319</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="auto_attack">
    <property name="geometry">
     <rect>
      <x>747</x>
      <y>532</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="background" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Resource/UI_Design/PVP_Settings_UI.png);</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>tone_Setting</zorder>
   <zorder>num_Setting</zorder>
   <zorder>move_Setting</zorder>
   <zorder>Name_Edit</zorder>
   <zorder>emoji_Setting</zorder>
   <zorder>NO4</zorder>
   <zorder>confirm</zorder>
   <zorder>NO2</zorder>
   <zorder>NO3</zorder>
   <zorder>NO1</zorder>
   <zorder>cancel</zorder>
   <zorder>auto_counter</zorder>
   <zorder>Provocative</zorder>
   <zorder>auto_attack</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../Resource/Gobang.qrc"/>
 </resources>
 <connections/>
</ui>
